docs/contributing/database-migrations/mssql.md

```diff
@@ -11,97 +11,72 @@ For instructions on how to apply database migrations, please refer to the
 
 :::
 
-## SQL database project
+:::tip
 
-We use a [SDK-style SQL project][MSBuildSQL] (`sqlproj`) to develop the database locally. This means
-we have an up-to-date representation of the database in `src/Sql`, and any modifications needs to be
-represented there as well. SDK-style SQL projects are still in preview the tooling is not yet
-available in Visual Studio. However it is available in [Visual Studio Code][vscode] and [Azure Data
-Studio][azureds] with the [SQL Database Projects][SDPE] extension, which provides schema comparison
-and more. You may also modify the `.sql` files directly with any text editor.
+We recommend reading [T-SQL Code Style](../code-style/sql.md) since it has a major impact in how we
+write migrations.
 
-To make a database change, start by modifying the `.sql` files in `src/Sql/dbo`. These changes will
-also need to be applied in a migration script. Migration scripts are located in
-`util/Migrator/DbScripts`.
+:::
 
-You can either generate the migration scripts automatically using the _Schema Comparison_
-functionality or by manually writing them. Do note that the automatic method will only take you so
-far and it will need to be manually edited to adhere to the code styles.
+## SQL database project
 
-For added safe guards we have automated linting and validation to ensure the SQL project is always
-up to date with the migrations.
```

Comment:
I prefer my section it explains the SQL project with links to references and how they relate to DbScripts.

Reply:
Grabbed most of it with some removals due to what exists today.